README.rst

```diff
@@ -5,7 +5,7 @@
 meson-python
 ============
 
-``meson-python`` is a Python build backend built on top of the Meson__
+``meson-python`` is a Python build back-end built on top of the Meson__
```

Comment:
I don't have a preference. I think I opted for back-end because it seemed like it was used more often than backend, or because I was annoyed by the spell-checker nagging me about the spelling. I agree that it is better to be consistent. I'll flip all occurrences to the latter spelling.

Reply:
Missed reverting this one instance I think?

Reply:
No, I just folded the change into the wrong commit. I missed one other instance though.